config/application.rb

```diff
@@ -34,5 +34,8 @@ class Application < Rails::Application
 
     config.i18n.available_locales = :"pt-BR"
     config.i18n.default_locale = :"pt-BR"
+
+    config.middleware.use Rack::Affiliates,
+      { param: 'aff', extra_params: [:campaign] }
```

Comment:
Can you move this to `config/initializers/rack_affiliates.rb`?

``` 
Rails.application.config.middleware.use(
  Rack::Affiliates, param: 'aff', extra_params: [:campaign]
)
```

We started without this rule, but I think it's best to avoid adding things directly to `application.rb`, the same way we avoid adding things to `rails_helper` and `spec_helper` ;)